content/uc-doc/upgrade/upgrade_notes.md

```diff
@@ -2,6 +2,14 @@
 title: Upgrade notes
 ---
 
+## 21.10 {#21-10}
+
+- The `wazo-dird` Google backend has migrated from using the deprecated Contacts API to the People API. To make valid requests, the Google Cloud application must have either the `https://www.googleapis.com/auth/contacts` or the `https://www.googleapis.com/auth/contacts.readonly` [permissions](https://developers.google.com/people/contacts-api-migration#read). Also, the Google Cloud application must enable the People API instead of the Contacts API; [here's how to do that](https://support.google.com/googleapi/answer/6158841?hl=en).
```

Comment:
Do you know if all users will have to "re-link" the Google account to accept the new permissions? If that's the case it should also be in the upgrade note.